index.html

```diff
@@ -205,10 +205,10 @@ <h4>Skills</h4>
       </section>
   <section id="contact-section">
     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-    <form action="#" id="form">
-      <input type="text" name="fullName" id="fullName"  placeholder="Full Name" required>
+    <form action="https://formspree.io/f/meqwdago" method="post" id="form">
+      <input type="text" name="fullName" id="fullName"  placeholder="Full Name" maxlength="30" required>
       <input type="email" name="email" id="email" placeholder="Email Address" required>
-      <textarea name="message" id="message" cols="30" rows="10"></textarea>
+      <textarea name="message" id="message" cols="30" rows="10" maxlength="500"></textarea>
       <button type="submit">Get in touch</button>
```

Comment:
- [ ] Dear @AsumVictor, I notice you have applied the hover effect for your button which is great. But we need the `active` effect also. Remember for a better design we need to follow the button interactions such as hover, active.